### ISSUE TITLE
Save dialog panics on macos

### ISSUE THREAD

**Issue by gfrancine** [gfrancine/mnote#10]
## Repro
1. Open a new editor without having any folder open
2. Save the file

```
thread 'main' panicked at 'Attempted to construct an Id from a null pointer', /Users/admin/.cargo/registry/src/github.com-1ecc6299db9ec823/objc_id-0.1.1/src/id.rs:62:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
libc++abi: terminating with uncaught foreign exception
```
## Repro with tauri
```ts
import * as dialog from "@tauri-apps/api/dialog";

// Doesn't crash
/* dialog.save({
  defaultPath: "~/Untitled"
}) */

// Crashes
dialog.save({
  defaultPath: "Untitled"
})
```

**Comment by gfrancine** [comment]
The fix should available in the next tauri version https://github.com/tauri-apps/tauri/pull/4028

**Comment by gfrancine** [comment]
fixed in tauri v1.0.0-rc.9
